src/index.html

```diff
@@ -10,13 +10,63 @@
 <body>
   <div>
     <div class="header">
-      <div class="header__logo"><img src="https://upload.wikimedia.org/wikipedia/commons/4/41/SEGA_logo.png"/></div>
+      <div class="header__logo"><img src="https://lh3.google.com/u/0/d/0BxpAmXfCG6jyY0k1MXJ5TUFBR0U=w1280-h711-iv1"/></div>
       <div class="header__nav">
         <div class="header__item"><a href="#" class="link">Home</a></div>
         <div class="header__item"><a href="#" class="link">Dome</a></div>
         <div class="header__item"><a href="#" class="link">Rome</a></div>
       </div>
     </div>
+
+  <div class="create">
+    <div class="progress">
+      <div class="progress__section">
+        <div class="progress__dot">
+          <span>1</span>
```

Comment:
This should have a class. Or even better, remove it and just type 1 within `.progress__dot`